### DISCUSSION TITLE
Auto Fit Text

### DISCUSSION THREAD

**Discussion by glenyen28** [ChurchApps/ChurchAppsSupport#321] [FreeShow] [answered]
Sometimes there is a long bible text scripture need to edit before you flash it on the screen. Can you make it auto fit when bible scripture text is too long. Thank you for your response.

**Comment by vassbo** [comment, marked as answer]
Have you tried enabling the "Auto" text size for the scripture template?

**Reply by glenyen28** [reply]
I see it there. Thank you and appreciate your response

**Reply by cyro-hub** [reply]
> Have you tried enabling the "Auto" text size for the scripture template?

What about making the text box grow instead of text resizing size

**Reply by vassbo** [reply]
> What about making the text box grow instead of text resizing size

That does not make sense.

**Reply by jerrytec** [reply]
> Have you tried enabling the "Auto" text size for the scripture template?

Where can I find the button for enabling Auto text size for scripture template? (Is it possible to see a screenshot please)

**Reply by jerrytec** [reply]
> I see it there. Thank you and appreciate your response

Where exactly please?

**Reply by vassbo** [reply]
@jerrytec On the right side:
![image](https://github.com/user-attachments/assets/9b4f51ae-c63c-47f7-af27-e8a449548d41)

**Reply by jerrytec** [reply]
Thank you I will check this out today

**Reply by jerrytec** [reply]
> @jerrytec On the right side:
> 
> ![image](https://github.com/user-attachments/assets/9b4f51ae-c63c-47f7-af27-e8a449548d41)

Thank you for your attention. I need some clarification regarding scripture auto size. For example, Genesis 1:1 fits the screen, but verse 2 of the same Genesis 1 does not until I covert to show, edit, and auto size before I can get out of that particular verse. Meanwhile, the preacher is wondering why the delay in showing a particular verse (you can understand their concerns, right?). My question is, is there a better way to make all scriptures autosize (i.e., fit the screen without having to do so manually)? Just like with songs, once I edit and make a song autosize and save it, it stays that way so I do not have to do it every time. I hope this makes sense and helps you understand the question. I made a video recording so you can see what I am talking about.

**Reply by vassbo** [reply]
Right click the "Scripture" template in the "Templates" drawer. Then enable the "Audo size" toggle. That should make the text auto size.
